src/WaipuData.cpp

```diff
@@ -187,8 +187,9 @@ void WaipuData::LoginThread()
                                                              PVR_CONNECTION_STATE_CONNECTED, "");
       m_nextLoginAttempt = std::time(0) + 60;
 
-      if (previousStatus != m_login_status)
+      if (previousStatus != m_login_status || m_lastUpdate < std::time(0) - 60*30)
```

Comment:
It is not related to the commit message. Why do we want to trigger Updates and why don't we leave it to Kodi?